docs/notebooks/synthesize_relational_database.ipynb

```diff
@@ -241,9 +241,7 @@
       "source": [
         "## Set up Relational Model and Create Project\n",
         "\n",
-        "During this step, you will be prompted to input your API key, which can be found in the [Gretel Console](https://console.gretel.ai/users/me/key).\n",
-        "\n",
-        "Relational Synthetics will use Amplify by default. Alternatively, you can set `gretel_model=\"actgan\"` or `gretel_model=\"lstm\"`. "
+        "During this step, you will be prompted to input your API key, which can be found in the [Gretel Console](https://console.gretel.ai/users/me/key)."
```

Comment:
We should strike this line, now that API key is entered at the beginning of the notebook

Reply:
Nice catch, thanks!